content/release-notes/2022-03/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "2022-03 release"
+description: ""
+lead: ""
+date: 2022-03-04T02:48:57+00:00
+draft: false
+toc: false
+resources:
+  - src:
+---
+
+### New features, enhancements, and bug fixes
+
+- Track and manage your work by marking the triaged state of each alert
+- Event reconciliation on pattern/logic changes
```

Comment:
Is it important to add that this change will be rolled out slowly for existing tenants?

Reply:
Good question. I have to admit I wasn't fully clear what this change involved. Please share more in Slack.